crates/trigger/src/cli.rs

```diff
@@ -43,6 +43,7 @@ where
     #[clap(
         name = APP_LOG_DIR,
         short = 'L',
+        env = APP_LOG_DIR,
```

Comment:
For an environment variable I'd consider `SPIN_LOG_DIR` (as Hippo used) rather than `APP_LOG_DIR`.  Reduces the risk of a clash, and makes it easier for someone poring through the environment to understand what the variable relates to.

Reply:
(to be clear `APP_LOG_DIR` is fine for the _name_ because that only gets used when printing help strings, at which point the user is strictly in the context of `spin up`... I am just wary of it as a name in the _global_ environment)